extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +141,509 @@ For example
+
+Both storage formats are appropriate for efficiently transmitting large quantities of property values.
+
+### Schema Definitions
+
+#### Overview
+
+Data types and semantic meanings of properties are provided by a schema, as defined in the [Cesium 3D Metadata Specification](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/specification/Metadata/) and summarized below.
+
+#### Schema
+
+*Defined in [schema.schema.json](./schema/schema.schema.json).*
+
+Top-level definitions for type and semantic information. The schema provides a set of *classes* and *enums* the asset can reference.
+
+A schema may be embedded in the extension directly or referenced externally with the `schemaUri` property. Multiple glTF assets may refer to the same external schema to avoid duplication. A schema is defined by an `EXT_mesh_features` extension attached to the glTF root object.
+
+> **Example:** A simple schema defining enums and classes.
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_mesh_features": {
+>       "schema": {
+>         "name": "Schema 001",
+>         "description": "An example schema.",
+>         "version": "3.5.1",
+>         "enums": { ... },
+>         "classes": { ... }
+>       }
+>     }
+>   }
+> }
+> ```
+
+#### Class
+
+*Defined in [class.schema.json](./schema/class.schema.json).*
+
+Template for features. Classes provide a list of properties with type and semantic information. Every feature must be associated with a class, and the feature's properties must conform to the class's property definitions. Features whose properties conform to a class are considered instances of that class.
+
+Classes are defined as entries in the `schema.classes` dictionary, indexed by an alphanumeric class ID.
+
+> **Example:** A "Tree" class, which might describe a table of tree measurements taken in a park. Property definitions abbreviated here, and introduced in the next section.
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_mesh_features": {
+>       "schema": {
+>         "classes": {
+>           "tree": {
+>             "name": "Tree",
+>             "description": "Woody, perennial plant.",
+>             "properties": {
+>               "species": { ... },
+>               "height": { ... },
+>               "diameter": { ... }
+>             }
+>           }
+>         }
+>       }
+>     }
+>   }
+> }
+> ```
+
+#### Class Property
+
+*Defined in [class.property.schema.json](./schema/class.property.schema.json).*
+
+Properties are defined abstractly in a class by their semantic meaning and data type, and are instantiated in a feature with specific values conforming to that definition. Properties support a richer variety of data types than glTF accessors or GPU shading languages allow, defined by `property.componentType`:
+
+- `componentType`
+  - **Type:** `string`
+  - **Required:** ✓ Yes
+  - **Allowed values:**
```

Comment:
I don't think this is too bad, but if we want to reduce the vertical space, what if we grouped them by type:

* INT8, INT16, INT32, INT64
* UINT8, UINT16, UINT32, UINT64
* etc?

Reply:
Done ✅